src/tests/backend/specs/api/pad.js

```diff
@@ -586,6 +588,25 @@ describe(__filename, function () {
     });
   });
 
+  describe('appendText', function () {
+    it('Append text to a pad Id', async function () {
+      const res = await agent.get(`${endPoint('appendText', '1.2.13')}&padID=${testPadId}&text=hello`)
+          .expect(200)
+          .expect('Content-Type', /json/);
+      assert.equal(res.body.code, 0);
+    });
+  });
+
+
+  describe('getText', function () {
+    it('Gets text on a pad Id at a given revision', async function () {
+      const res = await agent.get(`${endPoint('getText')}&padID=${testPadId}&rev=3`)
```

Comment:
Rather than add a new rev 4 and read rev 3, why not just read rev 2?

Reply:
My logic was that I just like to know that it was this test that made the change.  In heinsight it would have been smarter to make a new pad and just to rev0 and rev1...   So yea, could prolly drop some logic here!